Expensify/App
# Feature: Fast search

### Details

https://github.com/Expensify/App/issues/6069#issuecomment-955887403

### Fixed Issues

$ https://github.com/Expensify/App/issues/6069

### Tests | QA Steps
1. Open newDot on web or desktop.
2. Used Shortcut keys to open the search page.
3. type something immediately after.
4. When the search page is shown, your search input should not be lost.
5. and you should see searched results based on the input.

### Tested On

- [x] Mobile Web (Not affected)
- [x] iOS (Not affected)
- [x] Android (Not affected)

### Screenshots


#### Web | Desktop


https://user-images.githubusercontent.com/24370807/141067794-08c8f867-f35a-404e-8117-6ca55b58837f.mp4

#### Mobile Web


#### iOS


#### Android

![Screenshot 2021-11-10 12:54:56](https://user-images.githubusercontent.com/24370807/141068280-a0a75d26-e569-48fc-8eb5-2d93d16557f4.png)
